model/src/main/scala/ch.epfl.scala.index.model/release/LanguageVersion.scala

```diff
@@ -72,12 +72,16 @@ object Scala3Version {
   def isValid(version: BinaryVersion): Boolean =
     version match {
       case MajorBinary(3) => true
-      case PreReleaseBinary(3, _, _, ReleaseCandidate(x)) => true
+      case MinorBinary(3, _) => true
+      case PatchBinary(3, _, _) => true
       case _ => false
     }
 
+  def FamilyParser[_: P]: P[Scala3Version] =
+    ("scala3" ~ End).map(_ => Scala3Version(MajorBinary(3)))
+
   def Parser[_: P]: P[Scala3Version] = {
-    BinaryVersion.Parser
+    FamilyParser | BinaryVersion.Parser
```

Comment:
Accept binary version or family (`scala3`)

Reply:
This is confusing because this `Parser` is also used to parse artifact names. So it means `cats-core_scala3` are valid Scala targets which is wrong.

I suggest to create a new `FamilyOrVersionParser` in `LanguageVersion`:

```scala
object LanguageVersion {
  def FamilyOrVersionParser[_: P]: P[LanguageVersion] = {
    (Scala3Version.FamilyParser | LanguageVersion.Parser) ~ End
  }
}
```

then use it in `SearchApi.parseScalaTarget`